pkg/cmd/cli/cmd/env.go

```diff
@@ -63,8 +63,8 @@ syntax.`
   // Update all replication controllers in the project to have ENV=prod
   $ %[1]s env replicationControllers --all ENV=prod
 
-  // Remove the enviroment variable ENV from a pod definition on disk and update the pod on the server
-  $ %[1]s env -f pod.json ENV-
+  // Remove the enviroment variable ENV from all deployment configs
```

Comment:
I'll make my example an addition then, tks.

----- Original Message -----
> > @@ -63,8 +63,8 @@ syntax.`
> >    // Update all replication controllers in the project to have ENV=prod
> >    $ %[1]s env replicationControllers --all ENV=prod
> >  
> > -  // Remove the enviroment variable ENV from a pod definition on disk and
> > update the pod on the server
> > -  $ %[1]s env -f pod.json ENV-
> > +  // Remove the enviroment variable ENV from all deployment configs
> 
> The example is demonstrating a use case where you are using a local copy of
> config to apply to the server.  It's still relevant.
> 
> ----- Original Message -----
> > > @@ -63,8 +63,8 @@ syntax.`
> > >    // Update all replication controllers in the project to have ENV=prod
> > >    $ %[1]s env replicationControllers --all ENV=prod
> > >  
> > > -  // Remove the enviroment variable ENV from a pod definition on disk
> > > and
> > > update the pod on the server
> > > -  $ %[1]s env -f pod.json ENV-
> > > +  // Remove the enviroment variable ENV from all deployment configs
> > 
> > Should not, or maybe make it an addition?
> > 
> > Seemed a more intuitive example of removal, since removing based on a
> > definition on file does not work in some cases. You can't reproduce that
> > using the sample-app processed files, for example.
> > 
> > ----- Original Message -----
> > > > @@ -63,8 +63,8 @@ syntax.`
> > > >    // Update all replication controllers in the project to have
> > > >    ENV=prod
> > > >    $ %[1]s env replicationControllers --all ENV=prod
> > > >  
> > > > -  // Remove the enviroment variable ENV from a pod definition on disk
> > > > and
> > > > update the pod on the server
> > > > -  $ %[1]s env -f pod.json ENV-
> > > > +  // Remove the enviroment variable ENV from all deployment configs
> > > 
> > > Why did you change this?
> > > 
> > > ---
> > > Reply to this email directly or view it on GitHub:
> > > https://github.com/openshift/origin/pull/2250/files#r30358202
> > 
> > 
> > ---
> > Reply to this email directly or view it on GitHub:
> > https://github.com/openshift/origin/pull/2250/files#r30360416
> 
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/openshift/origin/pull/2250/files#r30360502